### PR TITLE
docs:  local-development

### PR DESCRIPTION
**Pull request by viceice**
## Changes

- Add a note for modern docker versions which are using buildx
- Fix the docker pnpm install command for windows

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: docs/development/local-development.md [hunk 1]
```diff
@@ -75,10 +75,15 @@ To build the correct docker image:
 docker build -f .devcontainer/Dockerfile -t renovatebot_local .
 ```
 
+More recent docker versions are using buildx by default, so you need to run the following to get the image loaded to the docker image store.
```
Comment (HonkingGoose) [inline docs/development/local-development.md:78]:
```suggestion
Starting from Docker Engine 23.0 and Docker Desktop 4.19, Docker uses Buildx by default.
So you must run the following command to get the image loaded to the docker image store:

```

Changes:

- Add newline to fix lint error
- Capitalize proper nouns like `Docker` and `Buildx`
- Say which versions use Buildx by default [^1]
- Split sentence into two parts

[^1]: https://docs.docker.com/build/architecture/ lists which versions use Buildx by default.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: docs/development/local-development.md [hunk 2]
```diff
@@ -75,10 +75,16 @@ To build the correct docker image:
 docker build -f .devcontainer/Dockerfile -t renovatebot_local .
 ```
 
+Starting from Docker Engine 23.0 and Docker Desktop 4.19, Docker uses Buildx by default.
+So you must run the following command to get the image loaded to the docker image store:
```
Comment (HonkingGoose) [inline docs/development/local-development.md:79]:
```suggestion
So you must run the following command to get the image loaded to the docker image store:

```

Comment (HonkingGoose) [inline docs/development/local-development.md:79]:
```suggestion
So you must run the following command to get the image loaded to the Docker image store:

```

Let's try that again. 😄

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: docs/development/local-development.md [hunk 3]
```diff
@@ -75,10 +75,16 @@ To build the correct docker image:
 docker build -f .devcontainer/Dockerfile -t renovatebot_local .
 ```
 
+Starting from Docker Engine 23.0 and Docker Desktop 4.19, Docker uses Buildx by default.
+So you must run the following command to get the image loaded to the Docker image store:
+```
```
Comment (HonkingGoose) [inline docs/development/local-development.md:80]: I don't know why GitHub strips my newlines. You'll have to put the newline in yourself. 😉

**Review by rarkins — APPROVED** [review]

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**